common/windows/delphi/ext/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# Keyman ext Makefile - 3rd party components
+#
+
+NOTARGET_SIGNCODE=yes
+
+!ifdef NODELPHI
+TARGETS=.virtual
+!else
+TARGETS=dcpcrypt #jedi mbcolor scfontcombobox cef4delphi
```

Comment:
OK, so those other components _haven't_ been moved yet.  I was a bit concerned when I saw the targets below that lacked corresponding moves.

Reply:
Yep, step by step ;-)